Extension/package.json

```diff
@@ -2194,6 +2194,7 @@
     "translations-import": "node ./tools/prepublish.js && gulp translations-import",
     "prepublishjs": "node ./tools/prepublish.js",
     "pretest": "tsc -p test.tsconfig.json",
+    "generatePackageHashes": "gulp generate-package-hashes",
```

Comment:
Should this run as part of the build process, so `integrity` will always get automatically updated before webpacked?  That might protect us from forgetting to update it manually.

Reply:
Yes, I also thought to run "generate-package-hashes" during the build process. But I don't know yet where in the build process to put it. Would this be in the yaml file?

Reply:
build.yml (and build.windows.yml), I think.  Sean could confirm.  It would go before `yarn run compile`, so it's already available when that command webpacks.

Reply:
GenerateHashes script is set to run before compiling in yml build files.